src/pages/products/prairietest/index.tsx

```diff
@@ -0,0 +1,239 @@
+import React from "react";
+import Head from "next/head";
+import Image, { ImageProps } from "next/image";
+import classnames from "classnames";
+
+import { PageBanner } from "../../../components/Banner";
+import { Heading } from "../../../components/Heading";
+import { BannerCTA } from "../../../components/CallToActionBanner";
+
+import diagramImage from "../../../lib/images/exam-management.png";
+import scannerImage from "../../../lib/images/swipe.png";
+import accommodationsImage from "../../../lib/images/pt-accommodations.png";
+import unifiedViewImage from "../../../lib/images/unified-view.png";
+import rescheduleImage from "../../../lib/images/reschedule.png";
+import proctorImage from "../../../lib/images/assign-proctor.png";
+import schedulerImage from "../../../lib/images/pt-reservation.png";
+import overridesImage from "../../../lib/images/overrides.png";
+import onlineFormatImage from "../../../lib/images/zoom-proctored.png";
+import cbtfFormatImage from "../../../lib/images/cbtf.jpg";
+import byodFormatImage from "../../../lib/images/byodinclass.png";
+
+import styles from "./index.module.scss";
+
+interface FeatureCardProps {
+  image: ImageProps["src"];
+  title: string;
+  children: React.ReactNode;
+}
+
+const FeatureCard: React.FC<FeatureCardProps> = ({
+  image,
+  title,
+  children,
+}) => {
+  return (
+    <article className={classnames("card overflow-hidden", styles.card)}>
```

Comment:
I first wanted the boarder (when created the cards), but then decided it would be better without them (and did not think it was bad to remove the boarder).

Reply:
I changed the styles in 4df1574.